### DISCUSSION TITLE
Is it possible to use WebTransport from HttpClient/ClientWebTransport for server-to-server BiDi communication?

### DISCUSSION THREAD

**Discussion by geekox86** [dotnet/runtime#87981] [General]
Greetings,

I am aware that .Net 7 has some support for WebTransport in the server side (Kestrel), but how about the client side (HttpClient or ClientWebTransport)?
Is it possible to use WebTransport from HttpClient/ClientWebTransport for server-to-server bidirectional communication?
If not, is it planned for future .Net?

Many thanks

**Comment by davidfowl** [comment]
https://github.com/dotnet/runtime/issues/43641
